pkgs/top-level/python-packages.nix

```
@@ -6794,6 +6794,8 @@ in {
   else
     callPackage ../development/python-modules/rsa/4_0.nix { };
 
+  rstcheck = callPackage ../development/python-modules/rstcheck { buildPythonPackage = pkgs.python3Packages.buildPythonPackage; inherit (pkgs) docutils; };
```

```suggestion
  rstcheck = callPackage ../development/python-modules/rstcheck {
    inherit (pkgs) docutils;
  };
```

If you want to limit it to python3 please use my above suggestion.